examples/travelapp-kogito-travel-agency.yaml

```diff
@@ -1,21 +1,25 @@
 apiVersion: app.kiegroup.org/v1alpha1
-kind: KogitoApp
+kind: KogitoBuild
 metadata:
   name: kogito-travel-agency
 spec:
-  enableEvents: true
-  enablePersistence: true
+  type: RemoteSource
+  gitSource:
+    uri: 'https://github.com/kiegroup/kogito-travel-agency-tutorial/'
+    contextDir: 06-kogito-travel-agency
+---
+apiVersion: app.kiegroup.org/v1alpha1
+kind: KogitoRuntime
+metadata:
+  name: kogito-travel-agency
+spec:
+  infinispan:
+    useKogitoInfra: true
+  kafka:
+    UseKogitoInfra: true
   envs:
-  - name: MP_MESSAGING_INCOMING_VISASAPPROVED_BOOTSTRAP_SERVERS
-  - name: MP_MESSAGING_INCOMING_VISASREJECTED_BOOTSTRAP_SERVERS
-  - name: MP_MESSAGING_OUTGOING_VISAAPPLICATIONS_BOOTSTRAP_SERVERS
-  - name: MP_MESSAGING_OUTGOING_KOGITO_PROCESSINSTANCES_EVENTS_BOOTSTRAP_SERVERS
-  - name: MP_MESSAGING_OUTGOING_KOGITO_USERTASKINSTANCES_EVENTS_BOOTSTRAP_SERVERS
-  build:
-    gitSource:
-      contextDir: 06-kogito-travel-agency
-      uri: "https://github.com/kiegroup/kogito-travel-agency-tutorial/"
-    # set your maven nexus repository
-    #mavenMirrorURL: http://nexus3-nexus.apps-crc.testing/repository/maven-public/
-    # enable to print maven transfer logs in console
-    #enableMavenDownloadOutput: true
+    - name: MP_MESSAGING_INCOMING_VISASAPPROVED_BOOTSTRAP_SERVERS
```

Comment:
I believe we can remove these envs now that Quarkus has been fixed.

Reply:
done